app/swagger/actions/apidocs/index.rb

```diff
@@ -0,0 +1,31 @@
+require 'swagger/blocks'
+
+module Base::Swagger::Actions::Apidocs::Index
+  include Swagger::Blocks
+
+  def self.included(action)
+    action.class_eval do
+      swagger_root do
+        key :swagger, '2.0'
```

Comment:
Is there a reason we chose version 2 of swagger / open api over version 3?

Reply:
This is by default. I will check if it works with version 3.